.github/CI/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# DEBUG
```

Comment:
This code is different than the one it replaces, where the two parts are done in separate build branches (debug and release)